### PR TITLE
Imu Driver for Dora

### PR DESCRIPTION
**Pull request by ChichengZZZ**
# Imu（Inertial Measurement Unit（惯性测量单元））

## Description

Description of the hardware:
IMU (Inertial Measurement Unit) is a sensor device that is used to measure and provide information about an object's acceleration, angular velocity (rotation rate), and sometimes its orientation with respect to the Earth's coordinate system. The IMU combines multiple sensors to provide a comprehensive set of data for understanding the motion and orientation of the object it is attached to.

What task does it do?
The primary task of an IMU is to capture and track the object's motion and orientation in 3D space. It is commonly used in various applications, including robotics, drones, virtual reality systems, motion capture, and navigation, where accurate motion data is crucial for performing specific tasks or providing valuable insights.

Which hardware is supported?
IMUs are available in various configurations, and the specific hardware components can vary based on the application's requirements. A typical IMU consists of the following sensors:

Accelerometer: Measures linear acceleration, allowing the IMU to detect changes in velocity and direction.

Gyroscope: Measures angular velocity, providing information about rotational motion.

### Installation

conda create --name new_env --file requirements.txt

### Launch Step

Connect the device successfully

### Development Guidelines

If your hardware requires that the development process include specific step, please document it, as to avoid unnecessary troubleshooting.

### Output

This driver outputs acceleration, angular velocity, device orientation, etc. in the form of a dictionary.

### CODE REVIEW

File: op_1/__pycache__/dora_assis.cpython-37.pyc
Comment (haixuanTao) [inline op_1/__pycache__/dora_assis.cpython-37.pyc:1]: Can we remove `__pycache__` files please?

File: op_2/op_2.py
Comment (haixuanTao) [inline op_2/op_2.py:1]: Why is this commited?

File: LICENSE
Comment (haixuanTao) [inline LICENSE:1]: Why is this deleted?

File: op_1/op_1.py
Comment (haixuanTao) [inline op_1/op_1.py:1]: This file is not used

**Comment by haixuanTao** [comment]
Can you put your development into the root path `vendors/imu/`. Thanks

File: op_1/__pycache__/dora_assis.cpython-37.pyc
Comment (ChichengZZZ) [reply op_1/__pycache__/dora_assis.cpython-37.pyc:1]: yes,but it doesn't matter

File: op_2/op_2.py
Comment (ChichengZZZ) [reply op_2/op_2.py:1]: Save it for later testing

File: op_1/op_1.py
Comment (ChichengZZZ) [reply op_1/op_1.py:1]: yes
